### DISCUSSION TITLE
ESP32 no SD card installer

### DISCUSSION THREAD

**Discussion by gabodiazjayme** [forkineye/ESPixelStick#506] [General]
--------- Instructions --------
Please provide answers directly below each section.
--------- Instructions ---------

**ESPixelStick Firmware Version**
4.0 beta 4

**Hardware Version**
esp32 wroom

**Binary release or compiled yourself?**
yes

**Operating System (and version)**
windows 10 pro

**Web Browser (and version)**
google chrome Versión 99.0.4844.74 (Build oficial) (64 bits)

**Access Point**

**Describe the bug**
after flashing, the serial output shows "no SD card installer"

**Comment by gabodiazjayme** [comment]
after flashing, the serial output shows "no SD card installer"

**Comment by MartinMueller2003** [comment]
Which platform did you flash and please give a link to your hardware so I can see what the needed pin definitions are.

**Reply by gabodiazjayme** [reply]
link esp https://naylampmechatronics.com/espressif-esp/384-nodemcu-32-30-pin-esp32-wifi.html
link lector de memoria https://naylampmechatronics.com/almacenamiento/104-modulo-lector-de-memoria-micro-sd-card.html

I would like to know which are the connection pins of my micro SD reader to ESP32, momentarily I have solved the problem using pins D18 (SCK) , D19 (MISO) and D23 (MOSI), I have changed pin D5 to D4 for the CS, I don't know if it is correct,

**Comment by MartinMueller2003** [comment]
Does your hardware have an SD card slot and is there a card inserted into the slot? If not, then this is a normal message.

**Comment by gabodiazjayme** [comment]
hello, i have added a reader module to the esp32, i have solved the problem by changing the SC from pin D5 to pin D4, now it works very well. thank you very much for your attention.




    On Saturday, April 2, 2022, 05:56:30 AM CST, Martin Mueller ***@***.***> wrote:  
 
 


Does your hardware have an SD card slot and is there a card inserted into the slot? If not, then this is a normal message.
